blockchain/block/blockstore.go

```diff
@@ -32,8 +32,9 @@ func (in *Store) ToProto() *iotextypes.BlockStore {
 	for _, r := range in.Receipts {
 		receipts = append(receipts, r.ConvertToReceiptPb())
 	}
+	// blob sidecar data are stored separately
 	return &iotextypes.BlockStore{
-		Block:    in.Block.ConvertToBlockPb(),
+		Block:    in.Block.ProtoWithoutSidecar(),
```

Comment:
this func only used by blockdao's `PutBlock` to convert a block to protobuf before writing to DB

another option is to create a `func (b *Block) WithoutSidecar() *Block`, but that incurs quite a lot of changes involving re-use of pointer to struct/interface etc., seems like an overkill, please refer to https://github.com/iotexproject/iotex-core/commits/nosidecar